Fix Decimal serialization

```
backend_1  | E                       AssertionError: assert Order(order_i...nfirmations=0) == Order(order_i...nfirmations=0)
backend_1  | E                         Omitting 20 identical items, use -vv to show
backend_1  | E                         Differing attributes:
backend_1  | E                         in_tx_amount: Decimal('50') != '50'
backend_1  | E                         out_tx_amount: Decimal('49.5') != '49.5'
```
See https://github.com/fincubator/booker/runs/834280334?check_suite_focus=true#step:3:636

--- a/tests/test_booker_gateway_api.py
+++ b/tests/test_booker_gateway_api.py
@@ -2,6 +2,7 @@ import pytest
 
 import asyncio
 from uuid import uuid4
+from decimal import Decimal
 import logging
 
 import zmq
@@ -28,7 +29,7 @@ new_order = Order(
     in_tx_from='',
     in_tx_to='',
     in_tx_hash='',
-    in_tx_amount=Amount('50'),
+    in_tx_amount=Decimal('50'),
     in_tx_created_at=0,
     in_tx_status=TxStatus.WAIT,
     in_tx_error=TxError.NO_ERROR,
@@ -37,7 +38,7 @@ new_order = Order(
     out_tx_from='finteh-usdt',
     out_tx_to='kwaskoff',
     out_tx_hash='',
-    out_tx_amount=Amount('49.5'),
+    out_tx_amount=Decimal('49.5'),
     out_tx_created_at=0,
     out_tx_status=TxStatus.WAIT,
     out_tx_error=TxError.NO_ERROR,

